### DISCUSSION TITLE
Latest GoPro mini 11 update  conflict

### DISCUSSION THREAD

**Discussion by DemetresP** [gopro/labs#361] [Q&A] [answered]
Hey all i had GoPro labs for mini 11 after the latest update,go pro labs was lost. I install it again but the camera went to previous firmware and keep asking for an update, which is a problem couse it doesn't let me see the preview on quick app. I ve tryied the logical things , reinstall and install everything, reset camera to default values but the problem is there. Shall i wait for a GoPro lab update, do i miss something? Tia

**Comment by dnewman-gpsw** [comment, marked as answer]
There is a Labs update for Mini coming.  It will be version 2.30.70.  It is being QA'd now.  So just ignore the updates messages from Quik, as that update will remove Labs.

**Reply by DemetresP** [reply]
Thanks for the reply, problem is that I can't see the preview, but i am glad an update for GoPro labs is coming

**Reply by rpossum** [reply]
yes, me too. The mini is essentially useless to me in this state b/c I need to see a preview. Looking forward to this fix!

**Reply by dnewman-gpsw** [reply]
This has been tracked as a App bug, and it is going to be fixed.  Although the new Labs will hopefully come out very soon, so two paths to a solution.

**Comment by dnewman-gpsw** [comment]
Mini Labs 2.30.70 is out today. https://gopro.github.io/labs/

**Reply by DemetresP** [reply]
That's great news. Thanks

**Reply by nammyhang** [reply]
My mini is going on a restart loop after the update. I cannot use anything and the mini has become so hot. Where can I find other firmware? I want to revert back.

**Reply by dnewman-gpsw** [reply]
Stock firmware is here: https://gopro.com/en/us/update/hero11-black-mini   Was the failure during the update, or something you did after the update?  If it was something you did, share the details, as there maybe a simply solution.

**Reply by YuriOVictor** [reply]
I have same problem. My hero 11 mini is going loop after scan the code for disable NR.
I updated to firmware labs and as soon as I scanned the first QR CODE (NR - 1) it started an uninterrupted sequence of reboots.
It turns on and shows ACTIVE : NR FRNT in the top display and then turns off and repeats the same thing.
I reset to factory settings to no avail. So I updated to the current stock firmware and it was back to normal.
When I tried again to reinstall LABs it already finished the update and the same problem came back, without even having a chance to scan another code or do anything.
Is there any chance to use LABs on this camera?

**Reply by dnewman-gpsw** [reply]
NR - 1 is not a valid code.  Can you share the full string of the command you scanned? 

Also if you can record any the files, they will have all the commands set on the metadata.  Share the metadata please using https://gopro.github.io/labs/control/metadata/

Final misc/qrlog.txt will have a log of what your camera is doing.  Share that too.

**Reply by YuriOVictor** [reply]
DEVC 
  DVID 1
  DVNM Global Settings
  VERS 8, 2, 2
  FMWR H22.03.02.30.70
  LINF LSU2081401402799
  CINF 217, 144, 125, 235, 233, 158, 201, 18, 206, 135, 121, 6, 132, 246, 186, 116
  CASN C3491124688322
  MINF HERO11 Black Mini
  MUID 3950874841, 315203305, 108627918, 1958409860, 0, 0, 0, 0
  CMOD 12
  MTYP 0
  OREN U
  DZOM N
  DZST 0
  SMTR N
  PRTN Y
  PTWB AUTO
  PTSH MED
  PTCL NATURAL
  EXPT AUTO
  PIMX 1600
  PIMN 100
  PTEV 0.0
  RATE .empty.
  SROT 15.0000
  EISE Y
  EISA HS EIS
  HCTL Off
  AUPT N
  APTO OFF
  AUDO AUTO
  BROD .empty.
  BRID 0
  PVUL F
  PRJT GPRO
  SOFF 0
  CLKS 0
  CDAT 0x0000000061D04672
  SCTM 0
  PRNA 10
  PRNU 0
  SCAP N
  CDTM 0
  DUST NO_LIMIT
  VRES 3840, 2160
  VFPS 30000, 1001
  HSGT OFF
  BITR STANDARD
  MMOD STEREO
  RAMP .empty.
  TZON 32767
  CLKC 0
  DZMX 0.0000
  CTRL Pro
  PWPR PERFORMANCE
  ORDP Y
  CLDP Y
  PIMD MANUAL
DEVC 
  DVID FOVL
  DVNM Large FOV
  ABSC 0.0000
  ZFOV 138.5928
  VFOV S
  MXCF x1x3x5
  MAPX 1.2100, -1.2758, 1.7752
  MYCF y1y3y5y1x2y3x2y1x4
  MAPY 0.9365, 0.4465, -0.7683, -0.3574, 1.1585, 0.3529
  PYCF r0r1r2r3r4r5r6
  POLY 0.0000, 1.8230, 0.1068, -0.6632, 0.3563, -0.0000, 0.0000
  ZMPL 0.6549
  ARUW 1.3333
  ARWA 1.7778
DEVC 
  DVID USRM
  DVNM User Metadata
  STRM 
    NR01 1
    FRNT 1
    EXPS 1
    USRD 1
DEVC 
  DVID HLMT
  DVNM Highlights

**Comment by rpossum** [comment]
Thank you David!!!
